src/components/ToastContainer.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { ToastContainer } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+
+const Toast = ({}: // add props here
+{
+  // add propes types here
+}) => (
```

Comment:
why the comments?

Reply:
As of now we do not need `props` in this. If in future we need props in this component, then this comment was added for that. Right now we don't need any props in this so I have removed these comments for now.